plasmapy/examples/plot_physics.py

```diff
@@ -57,4 +57,79 @@
     formulary.plasma_frequency(electron_concentration))
 plt.xlabel("Electron Concentration (m^-3)")
 plt.ylabel("Langmuir Wave Plasma Frequency (rad/s)")
+plt.grid()
+plt.show()
+
+############################################################
+# Let's compare our data to real-world data of plasma devices
+
+# Our 'Data'
+print("---Our Data---")
+print("n_e: " + str(electron_concentration))
+print("T_e: " + str(electron_temperature))
+print("B: " + str(B))
+plt.scatter(electron_concentration, formulary.plasma_frequency(electron_concentration))
+
```

Comment:
The second plot seems to be missing the "theoretical" curve:
```suggestion
plt.plot(n_e, formulary.plasma_frequency(n_e))

```